public/javascripts/songlistTable.js

```diff
@@ -23,16 +25,44 @@ var songlistTableComponent = (function() {
         })
     }
 
+    var addToPlaylist = function(name, id) {
+        m.request({
+                method: 'POST',
+                url: '/user/library',
+                data: {
+                    name: name,
+                    id: id
+                }
+            })
+            .then(function(data) {
+                if (data.url) {
+                    window.location.href = data.url
+                }
+            })
+    }
+
     var inLibrary = m.prop(currentInLibrary)
+    var playlistName = m.prop()
     var songlistTable = {
         view: function() {
-            // console.log(inLibrary())
             return [
                 m('table.table', [
                     m('thead', [
                         m('th', 'Title'),
                         m('th', 'Author'),
-                        m('th')
+                        m('th'),
+                        m('th', [
+                          m('select.selectpicker', {
+                              title: 'Choose Your Playlist',
+                              onchange: function() {
+                                  playlistName(this.value)
```

Comment:
does this.value work? I thought you tend to use `m.withAttr('value', function (value) {playlistName(value)})`